zarnitsa/DataAugmenterNLP.py

```diff
@@ -166,18 +174,20 @@ def get_most_similar(word, topn=topn):
             by_similarity = sorted(
                 queries, key=lambda w: word.similarity(w), reverse=True
             )
-
+            print([w.lower_ for w in by_similarity[:5]])
```

Comment:
we still haven't logger, y?(
maybe we should use loguru? it seems quite easy to start with

Reply:
it's not needed , this print in unwanted